models/aeif_cond_alpha.nestml

```diff
@@ -86,11 +86,11 @@ neuron aeif_cond_alpha_neuron:
 
     # Impulse to add to DG_EXC on spike arrival to evoke unit-amplitude
     # conductance excursion.
-    PSConInit_E nS/ms = 1.0 nS * e / tau_syn_ex
+    PSConInit_E nS/ms = nS * e / tau_syn_ex
```

Comment:
Actually, @Silmathoron, this would leave us with "e / tau_syn_ex * 1.0 nS". I would not be too comfortable with that without adding parents. i.e. "(e / tau_syn_ex ) * 1.0 nS" I am not certain these are necessarily improvements.